3.6/Modules/build_encoding.py

```diff
@@ -140,9 +145,12 @@ def get_len(x):
 
     for i in range(X_mat.shape[0]):
         es = encoded_mols[i].split("-")
+
         for j in range(X_mat.shape[1]):
             if j < len(es):
                 e = np.asarray([int(c) for c in es[j]])
+                if not len(e): continue
```

Comment:
This is one of the confusing bits.  You can't assign an empty list to a numpy array.  However, I'm not sure why this is happening in the first place.  I'm assuming that some fragmentations are returning nothing and are still being added somehow.